# HG changeset patch
# User Catalin Codreanu <codreanu.catalin@gmail.com>
# Date 1372632704 -7200
#      Mon Jul 01 00:51:44 2013 +0200
# Node ID dc24773d1acc26e56e79603ab9229dcd804d995d
# Parent  944ade6e7f66cc0ea985e09b72a16941bd5989df
* fr_FR.ts: updated french translation by Catalin

diff --git a/libgui/languages/fr_FR.ts b/libgui/languages/fr_FR.ts
--- a/libgui/languages/fr_FR.ts
+++ b/libgui/languages/fr_FR.ts
@@ -39,58 +39,58 @@
     <message>
         <location line="+1"/>
         <source>persistent</source>
         <translation>persistant</translation>
     </message>
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.cc" line="+64"/>
         <source>foreground</source>
-        <translation type="unfinished"></translation>
+        <translation>avant plan</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>background</source>
-        <translation type="unfinished"></translation>
+        <translation>arrière plan</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>selection</source>
-        <translation type="unfinished"></translation>
+        <translation>sélection</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>cursor</source>
-        <translation type="unfinished"></translation>
+        <translation>curseur</translation>
     </message>
 </context>
 <context>
     <name>QTerminal</name>
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.h" line="+116"/>
         <source>Copy</source>
-        <translation type="unfinished">Copier</translation>
+        <translation>Copier</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Paste</source>
-        <translation type="unfinished"></translation>
+        <translation>Coller</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Clear All</source>
-        <translation type="unfinished"></translation>
+        <translation>Tout Effacer</translation>
     </message>
 </context>
 <context>
     <name>QWinTerminalImpl</name>
     <message>
         <location filename="../qterminal/libqterminal/win32/QWinTerminalImpl.cpp" line="+1451"/>
         <source>copied selection to clipboard</source>
-        <translation type="unfinished"></translation>
+        <translation>sélection copiée vers le clipboard</translation>
     </message>
 </context>
 <context>
     <name>documentation_dock_widget</name>
     <message>
         <location filename="../src/documentation-dock-widget.cc" line="+34"/>
         <source>Documentation</source>
         <translation>Documentation</translation>
@@ -513,17 +513,17 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
         <location line="+107"/>
         <source>Rename file/directory</source>
         <translation>Rénommer fichier/répertoire</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Rename file/directory:
 </source>
-        <translation>Rénommer fichier/répertoire
+        <translation>Rénommer fichier/répertoire:
 </translation>
     </message>
     <message>
         <location line="+0"/>
         <source>
  to: </source>
         <translation>
  en : </translation>
@@ -804,17 +804,17 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
     <message>
         <location line="+15"/>
         <source>File name/location</source>
         <translation>Nom de fichier/chemin</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>File contents</source>
-        <translation type="unfinished">Contenu du fichier</translation>
+        <translation>Contenu du fichier</translation>
     </message>
     <message>
         <location line="+99"/>
         <source>Searching...</source>
         <translation>Recherche en cours...</translation>
     </message>
     <message>
         <location line="+32"/>
@@ -1473,44 +1473,44 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
     <message>
         <location line="+54"/>
         <source>Port:</source>
         <translation>Port :</translation>
     </message>
     <message>
         <location line="-27"/>
         <source>Username:</source>
-        <translation>Nom d&apos;utilisateur</translation>
+        <translation>Nom d&apos;utilisateur :</translation>
     </message>
     <message>
         <location line="+37"/>
         <source>Password:</source>
         <translation>Mot de passe :</translation>
     </message>
     <message>
         <location filename="../src/settings-dialog.cc" line="+69"/>
         <location line="+4"/>
         <location line="+334"/>
         <source>System setting</source>
         <translation>Reglages système</translation>
     </message>
     <message>
         <location line="-268"/>
         <source>IBeam Cursor</source>
-        <translation type="unfinished"></translation>
+        <translation>Curseur Trait Vertical</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Block Cursor</source>
-        <translation type="unfinished"></translation>
+        <translation>Curseur Block</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Underline Cursor</source>
-        <translation type="unfinished"></translation>
+        <translation>Curseur Tiret Bas</translation>
     </message>
     <message>
         <location line="+129"/>
         <source>Difference to the default size</source>
         <translation>Difference avecla taille par défaut</translation>
     </message>
     <message>
         <location line="+6"/>
@@ -1641,37 +1641,37 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
     <message>
         <location filename="../src/workspace-view.cc" line="+47"/>
         <source>Workspace</source>
         <translation>Espace de travail</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>View the variables in the active workspace.</source>
-        <translation>Liste des variables dans l&apos;espace de travail</translation>
+        <translation>Liste des variables dans l&apos;espace de travail.</translation>
     </message>
     <message>
         <location line="+75"/>
         <source>Copy</source>
         <translation>Copier</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Rename</source>
         <translation>Renommer</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Only top-level symbols may be renamed.</source>
-        <translation>Seuls les variables de plus haut niveau peuvent etre renommés</translation>
+        <translation>Seuls les variables de plus haut niveau peuvent etre renommés.</translation>
     </message>
     <message>
         <location line="+125"/>
         <source>View the variables in the active workspace.&lt;br&gt;</source>
-        <translation>Voir les variables dans l&apos;espace de travail. &lt;br&gt;</translation>
+        <translation>Voir les variables dans l&apos;espace de travail.&lt;br&gt;</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Colors for the storage class:</source>
         <translation>Couleurs pour le type de stockage :</translation>
     </message>
 </context>
 </TS>
